### PR TITLE
Use Ditto v4 as a peer dependency in Package.swift

### PR DESCRIPTION
**Pull request by okdistribute**
- [x] Use 4.0.0 when v4 is released.
- [x] Update presence viewer code from the mono repo because it had breaking changes in v4

Do not merge!

### CODE REVIEW

**Review by texasRanger09 — APPROVED** [review]

**Comment by okdistribute** [comment]
Coordinate with @thombles and @phatblat around this

**Comment by phatblat** [comment]
I converted this PR to a draft since we're waiting until after the LA meetup to release v4.

**Comment by kndoshn** [comment]
> Coordinate with @thombles and @phatblat around this. We should also update the presence viewer code for v4

@thombles @phatblat @okdistribute 
Do you recall what change we wanted to make for the presence viewer?

**Comment by thombles** [comment]
No further changes are required as far I know. I believe the job is to to build the monorepo version of the presence viewer (at the v4 tag) and update the web resources for the presence viewers in this repo. This can be published as v4 and will be compatible with Ditto SDK v4.

File: Package.swift [hunk 1]
```diff
@@ -23,8 +23,7 @@ let package = Package(
             targets: ["DittoDiskUsage"]),
     ],
     dependencies: [
-        // Ditto.diskUsage was added in 3.0.1
-        .package(url: "https://github.com/getditto/DittoSwiftPackage", from: "3.0.1"),
+        .package(url: "https://github.com/getditto/DittoSwiftPackage", from: "4.0.0-beta1"),
```
Comment (radazzouz) [inline Package.swift:26]: v4 is out. We should change this.

```suggestion
        .package(url: "https://github.com/getditto/DittoSwiftPackage", from: "4.0.0"),
```

**Review by radazzouz — APPROVED** [review]

**Review by magicwave — APPROVED** [review]

**Comment by magicwave** [comment]
Closing: v4.4.0 release specifies v4.0.0 in SPM package manifest

**Pull request closed without merging**